wallet/dapps-communication.md

```diff
@@ -411,9 +416,246 @@ required:
  "publicKey": "P12..."
 }
 ```
+</td>
+</tr>
+</tbody>
+</table>
+
+#### Rolls
+
+##### Sell
+
+<table>
+<thead>
+<tr>
+<th>Direction</th>
+<th>Type</th>
+<th>Format</th>
+<th>Example</th>
+</tr>
+</thead>
+<tbody>
+
+<tr>
+<td>Webpage to extension</td>
+<td><code>account.sellRolls</code></td>
+<td>
+
+```yaml
+type: object
+properties:
+  fee:
+    type: string
+    format: BigInt
+  amount:
+    type: string
+    format: BigInt
+required:
+  - fee
+  - amount
+```
+
+</td><td>
+  
+```json
+{
+ "fee": "12000000",
+ "amount": "10"
+}
+```
+
+</td>
+</tr>
+
+
+
+<tr>
+<td>Extension to webpage</td>
+<td><code>account.sellRolls.response</code></td>
+<td>
+
+```yaml
+type: object
+properties:
+  operationId:
+    type: string
+    format: base58check
```

Comment:
The base58check encoding is partialy true. there is a "O" prefix.

in any case I propose to remove this format because this is not the prupose of this standard.
```suggestion
```

Reply:
fixed